src/content/self-hosted/administration/certificates.mdx

```diff
@@ -32,8 +32,13 @@ ingress-nginx:
       default-ssl-certificate: $(POD_NAMESPACE)/your-ssl-certificate-secret
 ```
 
-In order to generate your certificates with [cert-manager](https://cert-manager.io/) you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
+If you are using [cert-manager](https://cert-manager.io/), you need to use a [DNS01](https://cert-manager.io/docs/configuration/acme/dns01/#delegated-domains-for-dns01) auth method in your [Issuer](https://cert-manager.io/docs/concepts/issuer/).
 Check the list of supported providers [here](https://cert-manager.io/docs/configuration/acme/dns01/#supported-dns01-providers) for more information.
+There are community guides for the following Cloud Providers:
+
+- [Amazon Route53](https://community.okteto.com/t/how-do-i-configure-okteto-with-cert-manager-and-aws-route53/273/2)
```

Comment:
Can we add these links to the cloud provider specific guides as well?

Reply:
On every installation guide there is a link to this doc, which has more information than just the cert manager installation. Would you still add a link to the community CM guide on each guide?
Merging this as it is, but I will create a different PR for adding the link in the provider's guides if you find it valuable

Reply:
Yes we should. Everyone needs this if they are not using their own certs, it's better to put it directly in the cloud provider guide rather than have them go over multiple documents.

Reply:
The things is that the community guide doesn't cover everything and it points back to this doc